### DISCUSSION TITLE
YASB 1.5.0 Update

### DISCUSSION THREAD

**Discussion by amnweb** [amnweb/yasb#82] [Announcements]
## **YASB Version 1.5.0 Update**

YASB version 1.5.0 has been relocated. The installer now installs YASB in the Program Files directory, making it available for all users.

Why this change?
- YASB is now installed in the Program Files directory and will be available for all users.
- YASB will be updated automatically for all users.
- Permissions issues will be resolved with Komorebi.

What does this mean for you?
- If you have version 1.4.3, you can run `yasbc update` to update to the latest version. Any other installation method will not work, and you will need to uninstall the previous version of YASB and install the new version.
- Configuration files remain the same and will be used by the new version of YASB.
- If you run `winget update` or use the .msi installer, you will install the new version of YASB, however the old version will not be removed and you will need to uninstall it manually, otherwise you will have two versions of YASB installed on your system which will not work correctly.

Possible issues:
- When you run command `yasbc update`, you may see an error message `The system cannot find the path specified.` this is a known issue and can be ignored. The update will still be successful.

**Comment by ninagrosse** [comment]
Installing to Program Files means, you need admin privileges. Is it still possible to install YASB for the current user only without admin privileges?

**Reply by amnweb** [reply]
Only previous version or clone repo and run as python in this case you don't need admin.
